ledger-core/virtual/integration/method_test.go

```diff
@@ -1351,3 +1351,64 @@ func Test_MethodCall_HappyPath(t *testing.T) {
 
 	mc.Finish()
 }
+
+func TestVirtual_Method_ForObjectWithMissingState(t *testing.T) {
+	t.Log("C5106")
+	mc := minimock.NewController(t)
+
+	server, ctx := utils.NewUninitializedServer(nil, t)
```

Comment:
If you don't need to mock runner you can do just utils.NewServer(nil,t)

Reply:
done